library/vault

```diff
@@ -1,8 +1,6 @@
 # maintainer: Jeff Mitchell <jeff@hashicorp.com> (@jefferai)
 
-0.6: git://github.com/hashicorp/docker-vault@c087a27b5ec93445e89dc46f25af973f114f1399 0.6.5
```

Comment:
Did you want to leave this as an alias for the newest `0.6.x` release (maybe add one for `0.7`)?  Up to you. :smile: